docs/src/main/paradox/client/index.md

```diff
@@ -1,6 +1,6 @@
 # Consuming Services (Client)
 
-Get our Quickstart project (available for [Scala](https://developer.lightbend.com/start/?group=akka&project=akka-grpc-quickstart-scala) and [Java](https://developer.lightbend.com/start/?group=akka&project=akka-grpc-quickstart-java)) to get started quickly.
+See the Akka gRPC Quickstart project (available for [Scala](https://developer.lightbend.com/start/?group=akka&project=akka-grpc-quickstart-scala) and [Java](https://developer.lightbend.com/start/?group=akka&project=akka-grpc-quickstart-java)) to get started quickly.
 
```

Comment:
we haven't migrated these quickstart projects yet - so seems best to admit that the Akka one is useful as a guide

Reply:
Fair enough, I guess we can always update this later when we migrate the docs.